pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools>=42", "meson>=0.60.0", "oldest-supported-numpy", "ninja"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+requires = ["setuptools>=42", "meson>=0.60.0", "ninja", "numpy>=2.0"]
```

Comment:
Does this mean you intend to support _only_ NumPy 2.x starting with the next release?

Reply:
No, this is the build-time requirements, which can differ from runtime requirements (specified in setup.py as `>=1.21`). Due to the ABI change in numpy2, we must now compile against numpy2 in order to support both numpy1 and numpy2 at runtime, see [here](https://numpy.org/doc/stable/dev/depending_on_numpy.html#numpy-2-abi-handling).

Reply:
ah, I see..  got it.   thanks ;)